test/merchant_test.rb

```diff
@@ -40,6 +40,11 @@ def test_revenue_reports_all_revenue_for_a_merchant
     assert_equal BigDecimal.new('1872.74'), result
   end
 
+  def test_revenue_with_date_reports_all_revenue_for_that_date
+    result = @merchant_repository.merchants[9].revenue('2012-03-24 15:54:10 UTC')
+    assert_equal BigDecimal.new('1872.74'), result
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.